apps/indexer/drizzle/schema.ts

```diff
@@ -587,6 +587,7 @@ export const addressReference = pgTable(
   table => [
     index("address_reference_address").using("btree", table.address.asc().nullsLast().op("text_ops")),
     index("address_reference_transaction_id").using("btree", table.transactionId.asc().nullsLast().op("uuid_ops")),
+    index("address_reference_address_transaction_id").using("btree", table.address.asc().nullsLast(), table.transactionId.asc().nullsLast()),
```

Comment:
comment: I'm not sure, but since we have both the sequelize and the drizzle schema, we should probably keep both synced https://github.com/akash-network/console/blob/main/packages/database/dbSchemas/base/addressReference.ts